ras_backstage/views/secure_messaging/send_message.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+from flask import jsonify, make_response, request
+from flask_restplus import fields, Resource
+from structlog import wrap_logger
+
+from ras_backstage import secure_messaging_api
+from ras_backstage.controllers import secure_messaging_controller
+from ras_backstage.decorators.jwt_decorators import get_jwt
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+message_details = secure_messaging_api.model('MessageDetails', {
+        'msg_from': fields.String(required=True),
+        'subject': fields.String(required=True),
+        'body': fields.String(required=True),
+        'thread_id': fields.String(),
+})
+
+
+@secure_messaging_api.route('/send-message')
+class SendMessage(Resource):
+    method_decorators = [get_jwt(request)]
+
+    @staticmethod
+    @secure_messaging_api.expect(message_details, validate=True)
+    @secure_messaging_api.header('Authorization', 'JWT to pass to secure messaging service', required=True)
```

Comment:
can we add this to the secure_message_api namespace?

Reply:
You should be able to but there's a bug with adding decorators to namespaces atm so it wont work

Reply:
Just realised we are already doing a check for the JWT, this decorator may not be needed. Can we possibly add tests for the secure messaging endpoints to make sure if the jwt is not provided it returns a 400/401?

Reply:
I think there's benefit in having both. The header decorator updates the swagger but doesn't reject reject requests missing the header. The get_jwt decorator actually deals with rejecting the request when it's missing.

Reply:
oh okay, so we'll need to add '.header('Authorization', 'JWT...', required=True)' to all endpoints going forward to show up on swagger